packages/material-ui-lab/src/LoadingButton/LoadingButton.js

```diff
@@ -51,22 +50,24 @@ const LoadingButtonRoot = styled(Button, {
       ...(styles.endIconLoadingEnd && {
         [`& .${loadingButtonClasses.endIconLoadingEnd}`]: styles.endIconLoadingEnd,
       }),
-      ...(styles.labelLoadingCenter && {
-        [`& .${loadingButtonClasses.labelLoadingCenter}`]: styles.labelLoadingCenter,
-      }),
     };
   },
-})({
+})(({ styleProps, theme }) => ({
+  transition: theme.transitions.create(['background-color', 'box-shadow', 'border-color'], {
+    duration: theme.transitions.duration.short,
+  }),
```

Comment:
add transition (exclude color) due to this glitch.

https://user-images.githubusercontent.com/18292247/121315541-a3e25900-c932-11eb-8b33-8ffe326408e0.mp4

Reply:
It still looks off:

![looks off](https://user-images.githubusercontent.com/3165635/121440143-a28f4b80-c987-11eb-9a9d-33699aab5397.gif)

The non centered cases are worse.